massgov/mayflower-patternlab
# DP-8674: Add woff2 variants of fonts and use in font face declarations

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Adds WOFF2 variants of our webfonts.  WOFF2 is about 30% smaller than WOFF, and using our font-face declarations, we can offer perfect backward compatibility. 


## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-8674)

## Steps to Test


1. View any pattern in Chrome, and check the network tab.  You should see WOFF2 files being downloaded instead of WOFF.  

## Screenshots
![screen shot 2018-04-19 at 5 33 19 pm](https://user-images.githubusercontent.com/654407/39019581-c9598870-43f7-11e8-9ed1-b3b75cc0012e.png)



## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
